core-primitives/stf-state-handler/src/test/sgx_tests.rs

```diff
@@ -129,18 +129,14 @@ pub fn test_ensure_subsequent_state_loads_have_same_hash() {
 	let (lock, initial_state) = state_handler.load_for_mutation(&shard).unwrap();
 	state_handler.write_after_mutation(initial_state.clone(), lock, &shard).unwrap();
 
-	let state_loaded = state_handler.load(&shard).unwrap();
+	let (_state_loaded, loaded_state_hash) = state_handler.load_clone(&shard).unwrap();
```

Comment:
This seems to be unused, how about?
```suggestion
	let (_, loaded_state_hash) = state_handler.load_clone(&shard).unwrap();
```